frontend/models/product-list/types.ts

```diff
@@ -157,12 +155,6 @@ export interface ProductListRelatedPostsSection {
    tags: string | null;
 }
 
-export interface ProductListFeaturedProductListSection {
-   type: ProductListSectionType.FeaturedProductList;
-   id: string;
-   productList: FeaturedProductList;
-}
-
 export interface FeaturedProductList {
```

Comment:
Should we get rid of this interface too?

Reply:
Nice catch! Removed :)